gpu-jobs.md

```diff
@@ -166,115 +89,170 @@ server, including:
 
 {:.gtable}
-  | Job type | Maximum runtime | Per-user limitation | Submit file flag | 
+  | Job type | Maximum runtime | Per-user limitation | 
   | --- |
-  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs | `+GPUJobLength = "short"` | 
-  | Medium | 24 hrs | 1/3 of CTHC GPU Lab GPUs |  `+GPUJobLength = "medium"` | 
-  | Long  | 7 days | 1 job with 1-2 GPUs |  `+GPUJobLength = "long"` | 
+  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs | 
+  | Medium | 24 hrs | 1/3 of CTHC GPU Lab GPUs |  
+  | Long  | 7 days | 1 job with 1-2 GPUs | 
+
+Each server in the GPU Lab has a single GPU slot reserved for interactive use. Interactive 
+jobs that use GPU Lab servers are restricted to using a single GPU and a 4 hour runtime. 
+
+These job types, runtimes, and per-user limitations are subject to change with
+short notice as the CHTC GPU Lab studies usage patterns.
 
-If you do not specify a job type, the `medium` job type will be used as the default.
+## 3. Submit Jobs Using the CHTC GPU Lab
 
-## C. Request Specific GPUs or CUDA Functionality (optional)
+The following options are needed in your HTCondor submit file in order 
+to access the GPUs in the CHTC GPU Lab: 
 
-If your software or code requires a specific version of CUDA, a certain
+- **Request GPUs**: All jobs that use GPUs must request GPUs in their submit file (along
+with the usual requests for CPUs, memory and disk).
+	```
+	request_gpus = 1
+	```
+	{: .sub}
+
+- **Request the CHTC GPU Lab**: To use CHTC's shared use GPUs, you need to opt-in to the GPU Lab. To 
+do so, add the
+following line to your submit file:
+	```
+	+WantGPULab = true
+	```
+	{: .sub}
+ 
+- **Indicate Job Type**: We have categorized three "types"
+of GPU jobs, characterized in the table [above]().  Indicate which job type you would 
+like to submit by using the submit file option below. 
+	```
+	+GPUJobLength = "short" 
+	# Can also request "medium" or "long"
+	```
+	{: .sub}
+	If you do not specify a job type, the `medium` job type will be used as the default. If 
+	your jobs will run in less than 12 hours, it is advantageous to indicate that they are 
+	"short" jobs because you will be able to have more jobs running at once. 
+
+- **Request Specific GPUs or CUDA Functionality**: If your software or code requires a specific version of CUDA, a certain
 type of GPU, or has some other special requirement, you will need to add
 a "requirements" statement to your submit file that uses one of the
-attributes shown above.
-
-If you want a certain class of GPU, use CUDACapability:
+attributes shown above. If you want a certain class of GPU, use CUDACapability:
+	```
+	requirements = (CUDACapability == 7.5)
+	```
+	{: .sub}
+	It may be tempting to add requirements for specific GPU servers or
+	types of GPU cards. However, when possible, it is best to write your
+	code so that it can run across GPU types and without needing the
+	latest version of CUDA.
+
+A complete sample submit file is shown here: 
```

Comment:
Should we link the GPU templates repository after this standalone example?

Reply:
Yes! this would be a good place for that as well. 

Also - do those need to be updated, given this new info?

Reply:
Yes, we can work on https://github.com/CHTC/templates-GPUs/issues/5 now that we have the opt in syntax and job types finalized.